src/main/scala/com/twitter/diffy/DiffyServiceModule.scala

```diff
@@ -70,6 +70,9 @@ object DiffyServiceModule extends TwitterModule {
   val skipEmailsWhenNoErrors =
     flag[Boolean]("skipEmailsWhenNoErrors", false, "Do not send emails if there are no critical errors")
 
+  var httpsPort =
+    flag[String]("httpsPort", "443", "Port to be used when using HTTPS as a protocol")
```

Comment:
This should be https.port instead of httpsPort in order to be consistent with the readme and how other ports are named.